unity/Assets/Scripts/Logic/TileMap.cs

```diff
@@ -15,12 +15,7 @@ public class TileMap : ITileMap<ITile>
 
         public ITile GetTile(Coordinate coordinate)
         {
-            if (IsEmpty(coordinate))
-            {
-                ITile tile = Tile.CreateInstance(EState.OnField, EType.Void, ENature.None, EBehaviour.None);
-                tile.Coordinate = coordinate;
-            }
-            return _axisDictionary[coordinate];
+            return IsEmpty(coordinate) ? null : _axisDictionary[coordinate];
```

Comment:
I remember what this originally was intended to do: if a coordinate was accessed, that "wasn't there", it would initialize a new one in order to not return null values.

Seeing as this wasn't implemented anyway and so far we haven't had any problesm with NullPointers, I think we can go forward with your change here.

Just wanted to mention it, so we might remember, if we ever run into any problems regarding this.